packages/@sanity/components/src/textinputs/TextInput.js

```diff
@@ -31,7 +31,7 @@ export default class DefaultTextInput extends React.PureComponent {
     onBlur: PropTypes.func,
     isClearable: PropTypes.bool,
     isSelected: PropTypes.bool,
-    isDisabled: PropTypes.bool,
+    disabled: PropTypes.bool,
```

Comment:
Renaming this is a potential breaking change. Have you searched for usages across other files in the repo?

Reply:
Found one use on SlugInput on form-builder. Updated

Reply:
Thanks!